package.json

```diff
@@ -12,6 +12,7 @@
     "@material-ui/pickers": "^3.2.10",
     "@reduxjs/toolkit": "^1.3.4",
     "algolia-places-react": "^1.5.1",
```

Comment:
There's still one naked reference to Algolia -- in `MissionForm` (for mission create) which should be dealt with (replaced with *AddressInput*) either in this pr or a separate.  Actually,  I have a [pr in on that form ](https://github.com/factn/resilience-app/pull/492)to switch it to the new DateTimeInput... I could expand the scope of that pr and get us off Algolia altogether?? @mat10tng @qdozaq

Reply:
yeah, lets make another issue for getting rid of address input together with Algolia instead

Reply:
https://github.com/factn/resilience-app/issues/494